commons-statistics-descriptive/src/test/java/org/apache/commons/statistics/descriptive/TestData.java

```diff
@@ -60,18 +67,33 @@ static Stream<double[]> testValues() {
         );
     }
 
-    static Stream<Arguments> testValuesNonFinite() {
+    /**
+     * Function which supplies data with non-finite values to test the <code>accept()</code> and <code>of()</code> methods.
```

Comment:
supplies a test data for a statistic in a single array. Each case will contain at least 1 non-finite value.